server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoError/index.ts

```diff
@@ -0,0 +1,172 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IMongoExceptionRetryRule } from "../IMongoExceptionRetryRule";
+class InternalErrorRule implements IMongoExceptionRetryRule {
+    private static readonly codeName = "InternalError";
+    match(error: any): boolean {
+        return error.code === 1
+            && error.codeName
```

Comment:
Is it possible for the `error.codeName` be something different than `"InternalError"` when `error.code` is `1`? Or will it always be `"InternalError"` for error code 1?

Reply:
As far as for now, it is always 1 as interanlError. The reason why I check both equality is I don't want to include some unknown exceptions. So for now I'd like to retry only on the errors I know. So if some `"UnknownError"` with `error.code` `1`, we can decide further retry rule